test/steps/v2_steps.py

```diff
@@ -3082,6 +3083,51 @@ def deserialize_json_to_contract(context):
     assert actual == context.abi_contract
 
 
+@given(
+    'a dryrun response file "{dryrun_response_file}" and a transaction at index "{txn_id}"'
+)
+def parse_dryrun_response_object(context, dryrun_response_file, txn_id):
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    dir_path = os.path.dirname(os.path.dirname(dir_path))
+    with open(
+        dir_path + "/test/features/resources/" + dryrun_response_file, "r"
+    ) as f:
```

Comment:
```suggestion
    file = Path.cwd() /  "test" / "features" / "resources" /  dryrun_response_file
    with open(file) as f:
```
(assuming a `from pathlib import Path` at the top)
... but I'm a `pathlib` fanboy

Reply:
there's also the `load_resource()` function which is probably better than my previous suggetion.

I.e.:

```python
drr_dict = json.loads(load_resource(dryrun_response_file, is_binary=False))
```